src/main/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExceptionMapper.java

```diff
@@ -163,7 +164,8 @@ private static Throwable getRootException(Throwable exception) {
     private ResponseEntity<Object> errorDetailsResponseEntity(Exception ex, HttpStatus httpStatus, String errorMsg) {
 
         LOG.error(HANDLING_EXCEPTION_TEMPLATE, ex.getMessage(), ex);
```

Comment:
Existing code issue tough...we should use @Slf4j Lombok annotation on class. and remove imports import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

and use log.info